case-lib/hijack.sh

```diff
@@ -76,11 +85,14 @@ function func_exit_handler()
             fi
         }
         sleep 1s
-        if pgrep "$loggerBin"; then
-            dloge "$loggerBin resisted pkill -TERM, using -KILL"
-            sudo pkill -KILL "$loggerBin"
-            exit_status=1
-        fi
+	for logtool in "$cavstoolBin" "$loggerBin"
+	do
+            if pgrep -f -n "$logtool"; then
```

Comment:
For example, when running cavstool.py, it will trigger 2 processes in process list. 
```
root       16635  0.0  0.0  13856  4748 pts/0    S+   08:34   0:00 sudo cavstool.py -l
root       16636  0.4  0.2  28744 16708 pts/0    S+   08:34   0:01 python3 /usr/local/bin/cavstool.py -l
```

With `-f` option, it will list all 2 processes 
```
pgrep -f cavstool.py
16635
16636
```
But it only list the child process with `-n` 
```
pgrep -f -n cavstool.py
16636
```

Reply:
That makes sense, remove `-n`.